aladdin/env.py

```diff
@@ -100,7 +100,9 @@ def set_repo_path(env_key: str, dir_key: str, repo_key: str, required: bool = Fa
             logger.error(err_message)
             return False
         *_, git_account, repo = remote.split("/")
-        repo_value = f"git@github.com:{git_account}/{repo}"
```

Comment:
I am unsure why this would result in a duplicated `git@github.com:`, so this is more of a workaround instead of a fix

Reply:
yeah, I just ran into this .. weird how it's happening